benchmark_runner/common/template_operations/templates/vdbench/internal_data/vdbench_pod_template.yaml

```diff
@@ -63,6 +63,10 @@ metadata:
   name: vdbench-{{ kind }}-{{ trunc_uuid }}-{{ scale }}
   {%- endif %}
   namespace: {{ namespace }}
+  annotations:
+    {%- if kind == 'kata' %}
+    io.katacontainers.config.hypervisor.virtio_fs_extra_args: '["-o","allow_direct_io"]'
```

Comment:
Do be certain to test this to ensure that it does what you expect.

Reply:
Yeah, I tested it.